### PR TITLE
LPS-195221 Don't render fragments with template errors when indexing

### PR DESCRIPTION
**Pull request by jkappler**
https://liferay.atlassian.net/browse/LPS-195221

### CODE REVIEW

**Comment by jkappler** [comment]
ci:test:relevant

File: modules/apps/layout/layout-impl/src/main/java/com/liferay/layout/internal/search/spi/model/index/contributor/LayoutModelDocumentContributor.java [hunk 1]
```diff
@@ -106,14 +94,8 @@ private void _addLayoutContentFields(Document document, Layout layout) {
...
 
-				String languageId = LocaleUtil.toLanguageId(locale);
-
-				if (layoutContentMap.containsKey(languageId)) {
-					continue;
-				}
-
 				layoutContentMap.put(
-					languageId,
+					LocaleUtil.toLanguageId(locale),
```
Comment (ruben-pulido) [inline modules/apps/layout/layout-impl/src/main/java/com/liferay/layout/internal/search/spi/model/index/contributor/LayoutModelDocumentContributor.java:98]: @jkappler 
Should we call `document.addText` directly here? 
We don't need layoutContentMap anymore, so we can get rid of it.

Comment (jkappler) [reply modules/apps/layout/layout-impl/src/main/java/com/liferay/layout/internal/search/spi/model/index/contributor/LayoutModelDocumentContributor.java]: done, thanks @ruben-pulido

**Comment by jkappler** [comment]
ci:stop

**Comment by ealonso** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**